swagger.yaml

```diff
@@ -4487,6 +4487,20 @@ definitions:
       architecture:
         type: string
 
+  gpu:
+    type: object
+    properties:
```

Comment:
Can you please add descriptions to the properties?  I know other parts of the inventory don't have, but for GPU not all of the properties are obvious to me (e.g., name, handle).

Reply:
I will add the descriptions and remove the handle, as it would represent the same information as businfo, only in a slightly different format. i.e. `pci@0000:00:02.0` vs `PCI:0000:00:02.0`.